### ISSUE TITLE
Runtimes need to be updated

### ISSUE THREAD

**Issue by hexagonrecursion** [flathub/com.github.Murmele.Gittyup#39]
Flatpak warns me that three runtimes you are using should be updated. I tried updating Gittyup, I tried uninstalling and reinstalling, but I still get this warning. It appears the latest version of `com.github.Murmele.Gittyup/x86_64/stable` that you published required outdated runtimes.

```
Info: runtime org.kde.Platform branch 5.15-21.08 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Platform and SDK
Info: applications using this runtime:
   com.github.Murmele.Gittyup

Info: runtime org.freedesktop.Platform.VAAPI.Intel branch 21.08 is end-of-life, with reason:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
   com.github.Murmele.Gittyup

Info: runtime org.freedesktop.Platform.GL.default branch 21.08 is end-of-life, with reason:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
   com.github.Murmele.Gittyup
```

**Comment by hexagonrecursion** [comment]
Possibly related #37

**Comment by Murmele** [comment]
yes currently I am blocked in the qt6 build, but when they are fixed I will directly go to qt6

**Comment by Murmele** [comment]
Gets fixed with #44
